docs/geofetch/README.md

```diff
@@ -56,6 +56,10 @@ geofetch -i GSE95654 --just-metadata
 geofetch -i GSE95654 --processed --just-metadata
 ```
 
+
+&#8291;**Note:** We ensure that GEOfetch is compatible with Unix, Linux, and Mac OS X. 
```

Comment:
I think it's just macOS now, MacOS X was [stopped in 2020](https://en.wikipedia.org/wiki/MacOS)